workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/utils/constants/filterOptions.ts

```diff
@@ -23,7 +23,7 @@ export const FILTER_MAP: {[key: string]: string} = {
 
 export const FILTER_OPTIONS: IProps = {
 	[JiraEnum.AFFECTED_VERSIONS]: ['2024.Q4', '2024.Q3', '2024.Q2', '2024.Q1'],
-	[JiraEnum.CATEGORIES]: ['Paas', 'Saas', 'Self-Hosted', 'Docker'],
+	[JiraEnum.CATEGORIES]: ['PaaS', 'SaaS', 'On Premise', 'Docker'],
```

Comment:
Should we sort these? @ryanschuhler thoughts?

Reply:
this is the order that will appear in the frontend, on the figma we have Docker > Self-Hosted (On Premise now) > PaaS > SaaS
i think we should change the order, just not sure if we should use sorted or the figma order (which kinda makes sense concrete > abstract)

Reply:
I asked Tiffany on the ticket.

Reply:
yes lets sort both Category and Issue Classification filters. Severity stays the same sorted in order of severity.

Reply:
Done!